ProteoformSuiteGUI/ProteoformSweet.cs

```diff
@@ -66,7 +68,12 @@ private void showForm(Form form)
         private void experimentTheoreticalComparisonToolStripMenuItem_Click(object sender, EventArgs e) { showForm(experimentalTheoreticalComparison); }
         private void experimentExperimentComparisonToolStripMenuItem_Click(object sender, EventArgs e) { showForm(experimentExperimentComparison); }
         private void proteoformFamilyAssignmentToolStripMenuItem_Click(object sender, EventArgs e) { showForm(proteoformFamilies); }
-        private void resultsSummaryToolStripMenuItem_Click(object sender, EventArgs e) { showForm(resultsSummary); }
+
+        private void resultsSummaryToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            ResultsSummary resultsSummary = new ResultsSummary(); //want to update/rerun every time page is opened
```

Comment:
Can we bind the lists to the ResultsSummary form instead of remaking the form each time we click this?